config/application.rb

```diff
@@ -49,6 +49,8 @@ class Application < Rails::Application
       when "production"
         config.eager_load = true
       end
+
+      ActiveRecord.legacy_connection_handling = false
```

Comment:
This is a class level constant should we suppress this of check for `Rails 7` before setting?  Per Rails 7 legacy support is deprecated.

Reply:
Rspec warns about the deprecation when run locally.

Reply:
Should we future proof this some?
```suggestion
      if ActiveRecord.respond_to?(:legacy_connection_handling)
        ActiveRecord.legacy_connection_handling = false
      end
```

Reply:
I think it would be `ActiveRecord.respond_to?(:legacy_connection_handling=)`

But sounds good to me :+1: